enums/IssueSelectionOptionCategory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file plugins/importexport/quickSubmit/enums/IssueSelectionOptionCategory.php
```

Comment:
This class name is pretty awkward and unclear. I think `IssueSelection` would be less of a mouthful. Also, just use the relative pathname in the repo for the `@file` tag (I know we don't do this very consistently yet).

Reply:
moved to core app .